README.md

```diff
@@ -47,6 +47,29 @@ Jinja2 Syntax
   name inside. For instance, always write it as `{{ variable_name_here }}`. The same goes if the value is
   an expression. `{{ variable_name | default('hiya, doc') }}`
 
+Python Guidelines
+===========
+
+* Review Ansible guidelines for
+[modules](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_best_practices.html)
+and [development](https://docs.ansible.com/ansible/latest/dev_guide/index.html).
+* Use [PEP8](https://pep8.org/).
+* File headers and functions should have comments for their intent.
+* Example playbooks are to live in the root directory, not in a
+separate subdirectory, and prefixed with `example-`.
+  Rationale: Ansible depends on particular structure, where directory
+names have special meanings. If one introduces a new directory, one
+wonders whether it also has some special meaning.
+* Use fully qualified role names in examples:
+`linux-system-roles.$ROLENAME` (with the Galaxy prefix).
+* Use RFC [5737](https://tools.ietf.org/rfc/rfc5737),
+[7042](https://tools.ietf.org/rfc/rfc7042) and
+[3849](https://tools.ietf.org/rfc/rfc3849) documentation IP addresses
+and MAC addresses in examples.
```

Comment:
There was a mix-up somewhere and some stuff related to playbooks got added to the Python section.  Add these lines to the _Ansible Best Practices_ section instead.

Reply:
Done. Take 2.